db/migrate/20160720100703_create_tasks.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTasks < ActiveRecord::Migration[5.0]
+  def change
+    create_table :tasks do |t|
+      t.string :title
+      t.text :description
+      t.date :due_date
+      t.boolean :status
```

Comment:
You can edit your migration if it's still only in your branch and then run `rails db:drop db:create db:migrate` to apply all migrations with the fixed one. This way you can avoid unnecessary migrations.